configure.ac

```diff
@@ -683,13 +683,13 @@ if test "$have_amd_opt" = yes && test "${enable_debug+set}" != "set" && test "$I
 		fi
 		SUBSTRCLANG='clang'
 		SUBSTRGCC='gcc'
-		if grep -q "$SUBSTRCLANG" <<<"$CC"; then
+		if echo "$CC" | grep -q "$SUBSTRCLANG"; then
 			if [[ -z "${AMD_ARCH}" ]]; then
```

Comment:
Aren’t `[[`/`]]` bashism as well?

Reply:
Yes, you're quite right. To be fair I spent exactly as much effort as needed to get things to run on a distro that used dash for sh and nothing more, and the clang branch was never hit. If I have copious free time I can check the whole file for bashisms (`shellcheck` does not pick up on this, for some reason).

Reply:
Hopefully these bashism stuffs can be found in Ubuntu GitHub Actions once implemented.